### ISSUE TITLE
ICP-CE installation fails

### ISSUE THREAD

**Issue by hassenius** [ibm-cloud-architecture/terraform-icp-vmware#7]
As reported by @svanga08 
When attempting to install using variable "icp_inception_image" = "ibmcom/icp-inception:2.1.0.3"

Ansible installer fails with 
```
odule.icpprovision.null_resource.icp-install (remote-exec): fatal: [192.168.2.29]: FAILED! => {"changed": true, "cmd": "echo 'placeholder' | docker login -u placeholder --password-stdin  || docker login -u placeholder -p placeholder ", "delta": "0:00:00.652211", "end": "2018-07-30 04:43:18.881510", "msg": "non-zero return code", "rc": 1, "start": "2018-07-30 04:43:18.229299", "stderr": "Error response from daemon: Get https://registry-1.docker.io/v2/: unauthorized: incorrect username or password\nWARNING! Using --password via the CLI is insecure. Use --password-stdin.\nError response from daemon: Get https://registry-1.docker.io/v2/: unauthorized: incorrect username or password", "stderr_lines": ["Error response from daemon: Get https://registry-1.docker.io/v2/: unauthorized: incorrect username or password", "WARNING! Using --password via the CLI is insecure. Use --password-stdin.", "Error response from daemon: Get https://registry-1.docker.io/v2/: unauthorized: incorrect username or password"], "stdout": "", "stdout_lines": []
```

Commenting out this section fixes the issue: https://github.com/ibm-cloud-architecture/terraform-icp-vmware/blob/master/icp-deploy.tf#L58-L62

**Comment by osowski** [comment]
I'm running https://github.com/ibm-cloud-architecture/terraform-icp-vmware/tree/master/other_examples/smallcluster_icp-ce with the same code included in the root Terraform template to support remote registries and am not hitting this issue.  I'm running through a couple different test scenarios now and will report back any findings, but 3 different scenarios yesterday led me unable to reproduce this issue.

**Comment by osowski** [comment]
I was able to reproduce this issue using the main Terraform template at the root of this repo.  The issue occurs when using the default coded value of the https://github.com/ibm-cloud-architecture/terraform-module-icp-deploy module in the template, which is currently set to `2.2.1`.  

Changing this value to the latest version of `2.3.3` and running `terraform init` to pull down that latest version will fix the issue and not cause the support for remote private registries to break the installation.

Closing this issue and we will update the default value in the main Terraform template to point to the latest version of the Terraform ICP module.
